cothority.go

```diff
@@ -115,5 +120,8 @@ func runServer(ctx *cli.Context) {
 // signature from each.
 func checkConfig(c *cli.Context) error {
 	tomlFileName := c.String("g")
-	return check.Config(tomlFileName)
+	if c.NArg() > 0 {
+		tomlFileName = c.Args().First()
+	}
+	return check.Config(tomlFileName, c.Bool("detail"))
```

Comment:
so the verification process is not in any lib or so ? It's "included" in the cosi, and the cothority depends on it ? mmhhh... We might want to do something else for later.